server.js

```diff
@@ -72,9 +71,40 @@ app.use(session({
     saveUninitialized: false
 }));
 
-app.listen(port, function() {
+/*
+    HTTPS protocol
+*/
+
+// Change to 443 in production
+const httpsPort = 8080;
+// Change to port 80 in production
+const httpPort = 8000;
+// Change to production website name
+const hostName = '127.0.0.1';
```

Comment:
this has to be manually changed? maybe it can pull form a properties file where other various configs can be, like port, hostname, path to cert or something along those lines.